Assets/Scripts/Player.cs

```diff
@@ -3,33 +3,32 @@
 
 public class Player : Entity
 {
-  private void FixedUpdate()
-  {
-    MovePlayer();
-  }
-
-  private void MovePlayer()
-  {
-    // Set keyboard
-    Keyboard kb = Keyboard.current;
+    private void FixedUpdate()
+    {
+        MovePlayer();
+    }
 
-    // Return if no keyboard detected
-    if (kb == null)
-      return;
+    private void MovePlayer()
+    {
+        // Set keyboard
+        Keyboard kb = Keyboard.current;
 
-    // If user presses UP arrow or W key, player moves up
-    if (kb.upArrowKey.IsPressed() || kb.wKey.IsPressed())
-      transform.position = new Vector2(transform.position.x, transform.position.y + speed * Time.deltaTime);
+        // Return if no keyboard detected
+        if (kb == null)
+            return;
 
-    // If user presses DOWN arrow or S key, player moves down
-    if (kb.downArrowKey.IsPressed() || kb.sKey.IsPressed())
-      transform.position = new Vector2(transform.position.x, transform.position.y - speed * Time.deltaTime);
+        // If user presses UP arrow or W key, player moves up
+        if (kb.upArrowKey.IsPressed() || kb.wKey.IsPressed())
+            transform.position = new Vector2(transform.position.x, transform.position.y + speed * Time.deltaTime);
 
-    if (kb.numpad0Key.IsPressed())
-    {
-      Vector3 initialCameraPos = new Vector3(0.0f, 0.0f, -10.0f);
-      CameraEffects.ShakeCamera(Camera.main, 0.5f, CameraEffects.Magnitude.Soft, initialCameraPos);
+        // If user presses DOWN arrow or S key, player moves down
+        if (kb.downArrowKey.IsPressed() || kb.sKey.IsPressed())
+            transform.position = new Vector2(transform.position.x, transform.position.y - speed * Time.deltaTime);
 
+        if (kb.oKey.IsPressed())
+        {
+            Vector3 initialCameraPos = new Vector3(0.0f, 0.0f, -10.0f);
+            CameraEffects.ShakeCamera(Camera.main, 0.5f, 0.1f, initialCameraPos);
+        }
```

Comment:
Oh I just realized this is attached to the Player script. Think we can attach the _CameraEffects.cs_ to the camera instead? That way when we expose the variables, it'll show up on the camera game object